projects/nae-example-app/src/app/views/demo-title-config/content/0/demo-title-config-content0-task-view.component.ts

```diff
@@ -10,12 +10,12 @@ import {
     NAE_TAB_DATA,
     NAE_TASK_VIEW_CONFIGURATION,
     SearchService,
-    TabbedTaskView,
     tabbedAllowedNetsServiceFactory,
     tabbedTaskViewConfigurationFactory,
     TaskViewService,
     ViewIdService,
-    ChangedFieldsService
+    ChangedFieldsService,
+    AbstractTabbedTaskViewComponent
```

Comment:
why were these imports changed? It seems unrelated to the rest of the PR

Reply:
it was not fixed from changing the Angular to version 13, this view is probably not used and I needed to fix it because of tests in example app